src/main/webapp/i18n/de/notification.json

```diff
@@ -77,8 +77,8 @@
                 "programmingTestCasesChanged": "Die Testfälle der Programmieraufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" wurden aktualisiert. Die Abgaben der Studenten sollten erneut gebaut und getestet werden, um neue Ergebnisse zu erstellen.",
                 "newManualFeedbackRequest": "Die Programmieraufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat eine neue manuelle Feedbackanfrage. Bitte bewerte das Feedback vor der Deadline.",
                 "illegalSubmission": "Die Aufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat illegale Abgaben von Studenten.",
-                "newExercisePost": "Die Aufgabe \"{{ placeholderValues.8 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat einen neuen Beitrag: \"{{ placeholderValues.2 }}\"",
```

Comment:
What does this have to do with the PR? Also, why did you change this from value 8 to 5? Do we have to change this in the native application, too?

Reply:
It is not related to the incorrect email subject, but an additional bugfix. But as it is really small I also included it here. There is no value 8. 5 is the correct value. So with 8 the notification is not displayed correctly.
I don't think you have to adjust the mobile applications if you implemented it based on the list: https://confluence.ase.in.tum.de/display/ArTEMiS/Notifications. There it correctly says the exercise title is 5.